packages/core/src/components/popover-menu/readme.md

```diff
@@ -32,20 +32,53 @@ use the `referenceEl` prop rather than the `selector` the referenced element can
 </TdsPopoverMenu>
 
 ```
+<br>
+### Controlled vs Uncontrolled
+
+#### Controlled mode
+The popover component will be shown or hidden based on a `trigger` prop and `selector` or `referenceEl` prop.
+
+Example:
+
+```html
+<button id="trigger">Open Popover</button>
+<tds-popover-menu [selector]="#trigger">
+  <tds-popover-menu-item>
+    <a href="#">Action</a>
+  </tds-popover-menu-item>
+</tds-popover-menu>
+```
+
+#### Uncontrolled mode
+The `open` prop can be used to control the visibility of the popover, meaning that the popover will be shown or hidden based on the `open` prop. Props `selector` or `referenceEl` are still needed in order to determine the position of the popover. 
+
+Example:
+
+```html
+<button id="trigger" onClick="{() => showPopover = !showPopover}">Open Popover</button>
+<tds-popover-menu id="popover" selector="#trigger" open="showPopover">
+  <tds-popover-menu-item>
+    <a href="#">Action</a>
+  </tds-popover-menu-item>
+</tds-popover-menu>
+
+```
+
```

Comment:
I think its the same here: https://github.com/scania-digital-design-system/tegel/pull/839/files#r1840354351